docs/concepts/pdp/factdb.mdx

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 11
+title: Local Datastore (FactDB)
+---
+
+# PDP Local Datastore (FactDB)
+
+FactDB is an embedded datastore to manage larges data volumes inside the PDP.
+
+It allows the PDP to store and query large volumes of data locally,
+reducing the latency and improving the performance of the PDP.
+
+:::info EAP
+This feature is available under Early Access Program.
+Please contact with us on [Slack](https://io.permit.io/slack) for more information.
+:::
+
+:::warning
```

Comment:
1. Add a title keyword after the warning, so it will replace the `DANGER` default title.
2. It is also not the best practice to include admonitions one after the other. Better if you can find a better placement for that. You can move it after the first paragraph in the next section.

Reply:
Moved the warning to the enable guide